<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionModify_Prisoner</class>
 <widget class="QWidget" name="actionModify_Prisoner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Prisoner</string>
  </property>
  <widget class="QLabel" name="ft_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ft.</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Birthdate_dateEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hair_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hair Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastName_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sentence_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sentence: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Born_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateOfConviction_dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="foto_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>22</horstretch>
     <verstretch>22</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="Sentence_textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Type sentence here 255 word max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPrisoner_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modify Prisoner</string>
   </property>
  </widget>
  <widget class="QLabel" name="height_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Hair_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="birthDate_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Birth date: </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateToBeLiberated_dateEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dateToBeLiberated_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date to be liberated: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastName_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="born_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Born in:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateOfConviction_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Conviction:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Height_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="EyeColor_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="eye_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eye Color: </string>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="modify_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addPrisoner_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionModify_Prisoner</receiver>
   <slot>modify_prisoner_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionModify_Prisoner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modify_prisoner_clicked()</slot>
  <slot>modify_image_clicked()</slot>
 </slots>
</ui>
